java/src/jmri/jmrit/display/layoutEditor/LayoutEditor.java

```diff
@@ -7144,7 +7144,7 @@ public Color getDefaultTrackColorColor() {
 
     @Nonnull
     public String getDefaultOccupiedTrackColor() {
-        return ColorUtil.colorToColorName(defaultOccupiedTrackColor);
+        return ColorUtil.colorToString(defaultOccupiedTrackColor);
```

Comment:
Do we have a test that makes sure colorToColorName and colorToString produce the same results for the named colors?

My concern is that the strings produced here MAY be stored in panel files, and we need to make sure we don’t break that.

Reply:
Thanks Paul, a justified concern, the replacement method doesn't covert to RGB strings.

Have changed to annotation, as far as I can make out, colorToString will only return null if null passed.